src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -34,46 +39,44 @@ extern double CCSN_CentralDens;
 bool Flag_CoreCollapse( const int i, const int j, const int k, const int lv, const int PID, const double *Threshold )
 {
 
-   bool Flag = false;
+   bool Flag      = false;
    bool MaxRefine = false;
 
    const double dh        = amr->dh[lv];
-   const double Center[3] = { amr->BoxCenter[0], amr->BoxCenter[1], amr->BoxCenter[2] };
    const double Pos   [3] = { amr->patch[0][lv][PID]->EdgeL[0] + (i+0.5)*dh,
                               amr->patch[0][lv][PID]->EdgeL[1] + (j+0.5)*dh,
                               amr->patch[0][lv][PID]->EdgeL[2] + (k+0.5)*dh  };
-
-   const double dx = Center[0] - Pos[0];
-   const double dy = Center[1] - Pos[1];
-   const double dz = Center[2] - Pos[2];
-   const double r  = sqrt(  SQR( dx ) + SQR( dy ) + SQR( dz )  );
+   const double Center[3] = { 0.5*amr->BoxSize[0], 0.5*amr->BoxSize[1], 0.5*amr->BoxSize[2] };
```

Comment:
use `amr->BoxCenter` instead of `0.5*amr->BoxSize` for `Center`. check other places.

Reply:
I have replaced them.